netplotbrain/plot.py

```diff
@@ -126,13 +126,27 @@ def plot(nodes=None, fig: Optional[plt.Figure] = None, ax=None, view: str = 'L',
     # Figure setup
     # Get number of non-legend rows
     nrows, view, frames = _nrows_in_fig(view, frames)
+    
+    #if subtitles is not specified, give each subplot its view name
+    if subtitles == []:
+        for i in range(0, nrows * frames):
+            subtitles.append('auto')
+    
+    if len(subtitles) != nrows * frames:
+        raise ValueError('Lenght subtitles must be the same as number of subplots')
```

Comment:
So if auto, and there are 5 frames going from LR, auto will make "Left", "", "Anterior", "", "Right". I think auto can still be a comment that goes into the function. Could you change this conditional to also allso subtitles='auto' to be allowed